### ISSUE TITLE
Binary file generated by `pkg` (of v0.4.1) does not work

### ISSUE THREAD

**Issue by Wesley-Young** [bedrock-stdhub/bedrock-stdhub#7]
The error log is as follows:
```
undefined:3
    compound: function () { [native code] },
                                    ^^^^

SyntaxError: Unexpected identifier
    at SizeOfCompiler.compile (C:\snapshot\dist\bedrock-stdhub.js)
    at ProtoDefCompiler.compileProtoDefSync (C:\snapshot\dist\bedrock-stdhub.js)
    at createProto (C:\snapshot\dist\bedrock-stdhub.js)
    at node_modules/prismarine-nbt/nbt.js (C:\snapshot\dist\bedrock-stdhub.js)
    at __require (C:\snapshot\dist\bedrock-stdhub.js)
    at Object.<anonymous> (C:\snapshot\dist\bedrock-stdhub.js)
    at Module._compile (pkg/prelude/bootstrap.js:1926:22)
    at Module._extensions..js (node:internal/modules/cjs/loader:1166:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:834:12)

Node.js v18.5.0
```
The generated raw `js` file can be run with Node 20. Indeedly this problem is related with `pkg` and Node 18. Since `pkg` has been deprecated, we have to find an alternative solution to get a single executable file, or, at least, a file (shell script?) that can be directly executed.

**Comment by Wesley-Young** [comment]
Update: Solved in pr #6 and #8. However, just re-implementing this part of logic is not a fundamental solution. The problem of `pkg` persists, and we have to find another tool to generate SEAs.

**Comment by Chrise2024** [comment]
> Update: Solved in pr #6 and #8. However, just re-implementing this part of logic is not a fundamental solution. The problem of `pkg` persists, and we have to find another tool to generate SEAs.

Now I am working on transplant this project to C#.
This is my repo https://github.com/Chrise2024/bedrock.NetHub
